### ISSUE TITLE
ERROR: no manifest file found

### ISSUE THREAD

**Issue by pvtai** [Huawei/Consumer#4]
After upgrade "com.huawei.agconnect" classpath to 1.2.0.300 as below I cannot run project, but downgrade to 1.0.0.300 it's working.

```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    
    repositories {
        google()
        jcenter()
        // huawei maven
        maven { url 'http://developer.huawei.com/repo/' }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.6.1'
        classpath 'com.huawei.agconnect:agcp:1.2.0.300'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        google()
        jcenter()
        // huawei maven
        maven { url 'http://developer.huawei.com/repo/' }
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}

```
Error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> ERROR: no manifest file found
```

**Comment by liaoxiaojia** [comment]
Hi, pvtai
Which Kit demo has this problem?

**Comment by tim4dev** [comment]
Try ```com.huawei.agconnect:agcp:1.2.1.301```

**Comment by androiddev1992** [comment]
Was facing the same issue with same version. By moving to version 1.3.0.300 did it for me

**Comment by nqnhat1988** [comment]
@liaoxiaojia I believe this issue is fixed with latest agc version. Could we close this?

**Comment by pvtai** [comment]
> Was facing the same issue with same version. By moving to version 1.3.0.300 did it for me

Thanks, resolved by 1.3.0.300
